batavia/types/Range.js

```diff
@@ -199,6 +199,18 @@ Range.prototype.__getitem__ = function(index) {
     }
 }
 
+
+Range.prototype.__add__ = function(other) {
+  var types = require('../types')
+  if (types.isinstance(other, types.Bool)) {
+    var msg = 'unsupported operand type(s) for +: '
```

Comment:
At column 1: [(no-multiple-empty-lines) More than 1 blank line not allowed.](http://eslint.org/docs/rules/no-multiple-empty-lines)